students/mitek_michal/lesson_03_functions/exponentiation.py

```diff
@@ -0,0 +1,13 @@
+
+def power(a, n):
+
+    if n == 0:
+        return 1
+    else:
+        return a * power(a, n - 1)
+
+
+a_input, n_input = float(input("Provide a ")), int(input("Provide n "))
```

Comment:
```
def power(a, n):

    if n == 0:
        return 1
    else:
        return a * power(a, n - 1)


def main():
    a_input, n_input = float(input("Provide a ")), int(input("Provide n "))
    print(power(a_input, n_input))
 

if __name__ == '__main__':
    main()

```

Please have look at this code. When you will run this module as a script program will be looking for ``` if __name__ == __main__ ``` statement in this case start main() function. You will have also a possibility to run this program in by import main() function. 

It will be great if you will add same changes in other solutions to lesson 3 :)